### DISCUSSION TITLE
For databases, is only SQLite and POSTGRES available?

### DISCUSSION THREAD

**Discussion by ray-wienand** [keystonejs/keystone#7597] [Questions] [answered]
Hi. I'm new to Keystone. 
This might be a dumb question. With Keystone 6 can I only use SQLite & Postgres?
I have started watching the ecommerce training by Wesbos and he uses MongoDb

**Comment by JamesBream** [comment, marked as answer]
In short, you can't use Mongo at the moment as it's a work in progress. Wes' course was using a pre-release version of Keystone v6 that I think had partial support (but was removed due to issues I guess). You can still follow the course along just the same with SQLite/Postgres though, shouldn't make a difference.

SQLite is fine for getting started locally, Postgres is what you'll need if you're wanting to deploy. I'd recommend [Railway](https://railway.app?referralCode=MAjLjK) as a really easy way to host Keystone and Postgres for free (it's what I'm using).

See for more info on when Mongo might be available again: https://github.com/keystonejs/keystone/discussions/7511

**Reply by ray-wienand** [reply]
Thanks @JamesBream. Appreciate the feedback.

**Reply by ray-wienand** [reply]
@JamesBream quick question. In Wes' course I see he has the backend and frontend separate. Is it still the same when I use Postgress and Railway?

**Reply by JamesBream** [reply]
@digital-inception Yep, use one repo in the same way Wes does, create a project in Railway and add two services, one for frontend and one for backend (technically one for your Postgres DB too). You can then point the services at `/frontend` and `/backend` respectively and they'll be deployed as two separate apps :)

A bit more detail here: https://docs.railway.app/deploy/monorepo#isolated-monorepo

**Reply by ray-wienand** [reply]
Thanks @JamesBream I appreciate your response. I'll give it a go

**Reply by cjmaret** [reply]
Hey @JamesBream, @digital-inception said that Wes has frontend and backend **separate**. But your answer says "yep, use one repo in the same way Wes does". But I think we all have two repos for our deployed projects, and switching from my fully set-up polyrepo to a monorepo is really not feasible at this point. 

Is there a way we can make Railway work for polyrepos?
